### PR TITLE
Move handleEndTransmission to its own file

### PR DESCRIPTION
**Pull request by TheMagnetar**
**When merged this pull request will:**
- Moves FUNC(handleEndTransmission) in sys_signal/XEH_PreInit.hpp to its own file
- To be merged before NO_DEDICATED #190

### CODE REVIEW

File: addons/sys_signal/fnc_handleEndTransmission.sqf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * Return Value:
+ * nil
+ *
+ * Example:
+ * ["ACRE_PRC152_ID_1", "handleEndTransmission", "ACRE_PRC152_ID_2"] call acre_sys_signal_fnc_handleEndTransmission
+ *
+ * Public: No
+ */
+ #include "script_component.hpp"
```
Comment (jonpas) [inline addons/sys_signal/fnc_handleEndTransmission.sqf]: Space at start.

File: addons/sys_signal/fnc_handleEndTransmission.sqf [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+ params ["", "", "_data"];
```
Comment (jonpas) [inline addons/sys_signal/fnc_handleEndTransmission.sqf]: Space at start copy pasting is not always nice. 😆

File: addons/sys_signal/fnc_handleEndTransmission.sqf [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ * Author: ACRE2Team
+ * Function called when radio transmission is finished (see sys_data/CfgAcreInterface.hpp).
+ *
+ * Arguments:
+ * 0:  Receiving radio ID <STRING> (Unused)
+ * 1:  Event (-> "handleEndTransmission") <STRING> (Unused)
+ * 2:  Radio ID with transmitting radio Id <ARRAY>
+ *
+ * Return Value:
+ * nil
```
Comment (jonpas) [inline addons/sys_signal/fnc_handleEndTransmission.sqf]: Just write `None`.

**Review by jonpas — CHANGES_REQUESTED** [review]

File: addons/sys_signal/fnc_handleEndTransmission.sqf (hunk 2 above)
Comment (TheMagnetar) [reply addons/sys_signal/fnc_handleEndTransmission.sqf]: xDDDDDDDD

**Review by jonpas — APPROVED** [review]

**Comment by jonpas** [comment]
Replaced by #216

**Pull request closed without merging**